boards/common/atmega/board.c

```diff
@@ -29,6 +29,8 @@
 
 void led_init(void);
 
+void __attribute__((weak)) led_init(void) {}
```

Comment:
This is nice. But I think not everyone is aware what `__attribute__((weak))` means. Mind to add a short comment, maybe something like "Add an empty led_init as fall back: if at link time another implementation of led_init not marked as weak (a.k.a. a strong symbol) is present, it will be linked in instead."

Reply:
Ok, added the comment.